pkg/cmd/dev/main.go

```diff
@@ -70,7 +70,7 @@ func init() {
 func runDev(ctx context.Context) error {
```

Comment:
In `init` above, we should add the following:
```
log.SetFlags(0)
log.SetPrefix("")
```

Surfacing timing info for the CLI's internal logging is probably not as useful.